modules/vpc-sc/main.tf

```diff
@@ -89,6 +93,93 @@ resource "google_access_context_manager_service_perimeter" "standard" {
           allowed_services   = each.value.enforced_config.vpc_accessible_services
         }
       }
+
+      dynamic "egress_policies" {
+        for_each = try(local.perimeter_egress_policies_enforced[each.key] != null ? local.perimeter_egress_policies_enforced[each.key] : [], [])
+
+        content {
+          dynamic "egress_from" {
+            for_each = try(var.egress_policies[egress_policies.value].egress_from != null ? [""] : [], [])
+
+            content {
+              identity_type = try(var.egress_policies[egress_policies.value].egress_from.identity_type, null)
+              identities    = try(var.egress_policies[egress_policies.value].egress_from.identities, null)
+            }
+          }
+          dynamic "egress_to" {
+            for_each = try(var.egress_policies[egress_policies.value].egress_to != null ? [""] : [], [])
+
+            content {
+              resources = try(var.egress_policies[egress_policies.value].egress_to.resources, null)
+
+              dynamic "operations" {
+                for_each = try(var.egress_policies[egress_policies.value].egress_to.operations, [])
+
+                content {
+                  service_name = try(operations.value.service_name, null)
+
+                  dynamic "method_selectors" {
+                    for_each = try([operations.value.method_selectors], [])
+
+                    content {
+                      method     = try(method_selectors.value.method, null)
+                      permission = try(method_selectors.value.permission, null)
```

Comment:
I was surprised by this. I see it supported on the provider resource, but it is not listed on the docs. Have you tested this?

Reply:
Found it on the docs, cool!